### PR TITLE
Launch Files Cleanup

### PR DESCRIPTION
**Pull request by mvoellmy**
### DONE
- [x] Removed enable_tty and added env. variable instead.
- [x] Removed unused and not working namespacing. Should be implemented newly when needed.
- [x] Simplified passing arguments to included launch files and added section detailing mechanics behind it:
https://github.com/esa-prl/ros2_best_practices/wiki/Parameters#passing-parameters-to-included-launch-files

### TODO
- [x] Test launch file on the rover
- [x] Format launch files with decided format.
- [x] Decide and document best practice approach to format of launch file.

### Add Action Styles
```
 start_node_cmd = Node(
    package='package_name',
    name='node_name',
    ...
  )

ld = LaunchDescription()
ld.add_action(start_node_cmd)
return ld
```
**vs.**
```
 start_node_cmd = Node(
    package='package_name',
    name='node_name',
    ...
  )

return LaunchDescription([
  start_node_cmd,
...])
```
**vs.**
```
return LaunchDescription([
  Node(
    package='package_name',
    name='node_name',
    ...
  )]
)
```

### CODE REVIEW

**Comment by mvoellmy** [comment]
I did not change the nav2_launch files since the nav stack is not working properly at the moment and needs more attention (might need to change to their update launch files anyway...).

File: launch/locomotion.launch.py [hunk 1]
```diff
@@ -45,11 +42,6 @@ def generate_launch_description():
         default_value=os.path.join(rover_config_dir, 'config', 'marta.yaml'),
         description='Full path to the ROS2 parameters file to use for all launched nodes')
```
Comment (maxehrhardt) [inline launch/locomotion.launch.py:43]:
```suggestion
        description='Full path to the ROS2 parameters file to use for all launched nodes.')
```

File: launch/locomotion.launch.py [hunk 2]
```diff
@@ -79,95 +71,89 @@ def generate_launch_description():
...
+        parameters=[configured_params]
+    )
+
+    joy_cmd = Node(
+        package='joy',
+        executable='joy_node',
+        name='joy_node',
+        remappings=[
+                ('joy', 'gamepad')
+        ],
```
Comment (maxehrhardt) [inline launch/locomotion.launch.py:92]:
```suggestion
        remappings=[('joy', 'gamepad')],
```

File: launch/locomotion.launch.py [hunk 3]
```diff
@@ -79,95 +71,89 @@ def generate_launch_description():
...
+        parameters=[configured_params],
+    )
+
+    simple_rover_locomotion_cmd = Node(
+        package='simple_rover_locomotion',
+        executable='simple_rover_locomotion_node',
+        name='simple_rover_locomotion_node',
+        remappings=[('/rover_motion_cmd', '/cmd_vel')],
+        # Parameters can be passed as dict or path to the .yaml
+        parameters=[configured_params]
```
Comment (maxehrhardt) [inline launch/locomotion.launch.py:117]:
```suggestion
        parameters=[configured_params],
```

File: launch/locomotion.launch.py [hunk 4]
```diff
@@ -79,95 +71,89 @@ def generate_launch_description():
...
+    )
+
+    stop_mode_cmd = Node(
+        package='locomotion_mode',
+        executable='stop_mode_node',
+        name='stop_mode_node',
+        remappings=[('/rover_motion_cmd', '/cmd_vel')],
+        # Parameters can be passed as dict or path to the .yaml
+        parameters=[configured_params]
```
Comment (maxehrhardt) [inline launch/locomotion.launch.py:126]:
```suggestion
        parameters=[configured_params],
```

File: launch/locomotion.launch.py [hunk 5]
```diff
@@ -79,95 +71,89 @@ def generate_launch_description():
...
+    )
+
+    ptu_control_cmd = Node(
+        condition=IfCondition(use_ptu),
+        package='ptu_control',
+        executable='ptu_control_node',
+        name='ptu_control_node',
+        parameters=[configured_params]
```
Comment (maxehrhardt) [inline launch/locomotion.launch.py:134]:
```suggestion
        parameters=[configured_params],
```

File: launch/platform_driver_ethercat.launch.py [hunk 6]
```diff
@@ -14,11 +16,11 @@ def generate_launch_description():
...
-    pd_config_file = LaunchConfiguration('pd_config_file')
+    pd_config_file_path = LaunchConfiguration('pd_config_file_path')
 
     # Launch declarations
-    declare_pd_config_file_cmd = DeclareLaunchArgument(
-        'pd_config_file',
+    declare_pd_config_file_path_cmd = DeclareLaunchArgument(
+        'pd_config_file_path',
         default_value=os.path.join(pd_config_dir, 'marta.yaml'),
         description='Full path to the ROS2 parameters file to use for all launched nodes')
```
Comment (maxehrhardt) [inline launch/platform_driver_ethercat.launch.py:25]:
```suggestion
        description='Full path to the ROS2 parameters file to use for all launched nodes.')
```

File: launch/platform_driver_ethercat.launch.py [hunk 7]
```diff
@@ -30,7 +32,7 @@ def generate_launch_description():
             name = 'platform_driver_ethercat_node',
             output = 'screen',
             # arguments = ['--ros-args', '--log-level', 'debug'],
```
Comment (maxehrhardt) [inline launch/platform_driver_ethercat.launch.py:34]: Is this needed?

**Pull request merged**